assets/scss/stripe.scss

```diff
@@ -1,5 +1,123 @@
-@include responsiveBase() {
-	.stripe{
+/*doc
+---
+title: Stripe
+---
+A `.stripe` class is rendered by the `<Stripe>` component to
+visually divide the page with borders or stripes of color.
+
+To align content within a stripe, nest children in a
+`<Bounds>` component.
+
+#### Stripe variants
+A class of `.stripe` is default; the following classes are
+optional variants:
+
+Class                | Description
+-------------------- | ------------------------------------------------------------
+`.stripe--inverted`  |  rendered by the `inverted` prop, applies the standard `$C_contentBGInverted` background color
+`.stripe--collection`|  rendered by the `collection` prop, applies a light gray background; for use with card collections
+`.stripe--red`       |  applies the standard `$C_red` background color
+`.stripe--purple`    |  applies the standard `$C_purple` background color
+`.stripe--plum`      |  applies the standard `$C_plum` background color
+`.stripe--black`     |  applies the standard `$C_black` background color
+`.stripe--photoHero` |  rendered by the `backgroundImage` prop, for hero sections with a photo (see below)
+*/
+
+%stripe,
+.stripe {
+	@include standardBorder();
+	background-color: $C_contentBG;
```

Comment:
It shouldn't be too much effort to go back and fix this. For the future I'll make sure to only include overrides